packages/monitoring-system/data-processor/test/unit/data-processors/mocks/octokit-request-parser.ts

```diff
@@ -0,0 +1,197 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+/**
+ * NOTE: This file is duplicated in gcf-utils/src but cannot be imported
+ * from there since it's not part of the same root directory
+ * TODO(asonawalla): Publish gcf-utils/src/octokit-request-parser as a separate
+ * module and import it here instead
+ */
```

Comment:
^ as this note mentions, this file is an exact copy of `gcf-utils/src/octokit-request-parser.ts` but I can't import it from there because it's not part of the root dir for this project.

Long term plan is to publish this as a separate module (or a collection of modules called `octotools`?) and import it from there. Until then, sorry about the duplication.